include/odp/api/spec/packet.h

```diff
@@ -1990,6 +1990,23 @@ int8_t odp_packet_shaper_len_adjust(odp_packet_t pkt);
  */
 void odp_packet_shaper_len_adjust_set(odp_packet_t pkt, int8_t adj);
 
+/**
+ * Get mark value from the odp_packet_t
+ *
+ * When a flow is configured with mark PMR, Upon matching the rule,
```

Comment:
As I commented earlier, details on how the classifier sets the mark is a bit out-of-place here. From packet point-of-view the mark is just metadata and it should suffice to just briefly mention here that classifier can set it.

The default value must be specified here as one can call this function for any packet, including those that never went through the classifier.

Reply:
OK. I will reword it next version. Please check the next version, if it is not suffice, You could share the exact text to replace.